### ISSUE TITLE
[FEATURE] - Addon outdated info

### ISSUE THREAD

**Issue by Kamian404** [Azeroth-Pilot-Reloaded/azeroth-pilot-reloaded#259]
### Description

At the moment, the player maybe wont know that the addon is outdated. It would be nice to have the functionality that APR knows it and informs the user that there's a newer version

### Solution

Add a check if the addon is outdated. If so, add an alert (show it only once) that there's a new version
